### PR TITLE
Add logos to JuMP-dev 2024 page

### PR DESCRIPTION
**Pull request by odow**
Logo looks like

image

### CODE REVIEW

**Comment by odow** [comment]
For now I've added the Google logo. It wasn't obvious whether we need internal sign-off before using, so we should check before merging this.

image

**Comment by mlubin** [comment]
What about merging the GERAD changes and and then figuring out the details for the Google logo separately?

File: _posts/2023-10-22-jump_dev_2024.md [hunk 1]
```diff
@@ -75,3 +77,8 @@ JuMP-dev will be hosted by [GERAD](https://www.gerad.ca/en), located in the
  * Joshua Pulsipher (UWaterloo)
 
 To contact the program committee, write to `jump-dev-2024@googlegroups.com`.
+
+## Sponsors
+
+<a href="https://www.gerad.ca/en" style="background-image:none;"><img style="height:60px;margin:30px" src="/assets/jump-dev-workshops/2024/Gerad_Logo_CMYK_ang_Med_Res.jpg" alt="GERAD logo"></a>
+<a href="https://opensource.google/" style="background-image:none;"><img style="height:60px;margin:30px" src="/assets/jump-dev-workshops/2024/logo_Google_FullColor_hdpi_830x271px.png" alt="Google logo"></a>
```
Comment (odow) [inline _posts/2023-10-22-jump_dev_2024.md:84]:
```suggestion
```

**Review by mlubin — APPROVED** [review]

File: assets/jump-dev-workshops/2024/logo_Google_FullColor_hdpi_830x271px.png
Comment (blegat) [inline assets/jump-dev-workshops/2024/logo_Google_FullColor_hdpi_830x271px.png:1]: Maybe remove this file for now ?

**Pull request merged**